config/routes.rb

```diff
@@ -13,6 +13,7 @@
   resources :listings, only: [:index]
   namespace :user, path: "/:slug" do
     resources :listings, except: [:index]
+    get '/dashboard', to: 'dashboard#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.